src/main/java/org/stellar/sdk/StrKey.java

```diff
@@ -42,6 +52,12 @@ public static AccountID encodeToXDRAccountId(String data) {
         return accountID;
     }
 
+    public static Optional<VersionByte> decodedVersionByte(String data) {
```

Comment:
```suggestion
    public static Optional<VersionByte> decodeVersionByte(String data) {
```
Can we rename this function for consistency with the other function names?